src/client/containers/QuizPage/QuizPage.component.js

```diff
@@ -1,14 +1,70 @@
-import React from 'react';
-import './QuizPage.styles.css';
-import PageHeader from '../../components/PageHeaderComponent/PageHeader';
-import QuestionComponent from '../../components/Question/QuestionComponent';
-import QuizAnswers from '../../components/QuizAnswers/QuizAnswers';
-import SideMenu from '../../components/sideMenubar/sideMenu';
-import Buttons from '../../components/Buttons/Buttons';
-import ProgressBar from '../../components/ProgressBar.components/ProgressBar';
+import React, { useState } from 'react';
+import PageHeader from '../../components/PageHeader/PageHeader.components';
+import QuestionComponent from '../../components/Question/Question.component';
+import QuizAnswers from '../../components/QuizAnswers/QuizAnswers.components';
+import SideMenu from '../../components/sideMenubar/sideMenu.component';
+import Buttons from '../../components/Buttons/Buttons.component';
+import ProgressBar from '../../components/ProgressBar/ProgressBar.components';
 import image1 from '../../assets/images/questionBackgrounds/question1background.png';
+import image2 from '../../assets/images/questionBackgrounds/question2background.png';
+import image3 from '../../assets/images/questionBackgrounds/question3background.png';
+import image4 from '../../assets/images/questionBackgrounds/question4background.png';
+import image5 from '../../assets/images/questionBackgrounds/question5background.png';
+import './QuizPage.styles.css';
 
+const QuizQuestions = [
+  {
+    question: 'When visiting a website,what is that you are most intrested in?',
+    image: image1,
+    isAgreementQuestion: false,
+    firstAnswer: 'How the website looks and how easy it is for the users',
+    secondAnswer: 'Logic behind how the website is built',
+    level: 20,
+  },
+  {
+    question:
+      'I can easily notice changes around me including people,culture,trends,etc.',
+    image: image2,
+    isAgreementQuestion: true,
+    firstAnswer: '',
+    secondAnswer: '',
+    level: 40,
+  },
+  {
+    question:
+      'When I have a list of tasks to do,I prefer to multitask rather than focusing on singular tasks one at a time',
+    image: image3,
+    isAgreementQuestion: true,
+    firstAnswer: '',
+    secondAnswer: '',
+    level: 60,
+  },
+  {
+    question: 'I can easily understan what someone else is going through',
+    image: image4,
+    isAgreementQuestion: true,
+    firstAnswer: '',
+    secondAnswer: '',
+    level: 80,
+  },
+  {
+    question: 'I work well under pressure',
+    image: image5,
+    isAgreementQuestion: true,
+    firstAnswer: '',
+    secondAnswer: '',
+    level: 100,
+  },
+];
 export const QuizPage = () => {
+  const [currentOn, setCurrentOn] = useState(0);
+  const handleQuestions = () => {
+    setCurrentOn((prev) => prev + 1);
+  };
+  const goToPrevious = () => {
+    setCurrentOn((prev) => prev - 1);
+  };
+  console.log(currentOn, QuizQuestions.length - 1);
```

Comment:
I think, You forgot to remove console here before pushing the code...!

Reply:
ya .. Thanks i will remove it..

Reply:
Thanks for @rajanibusani reviewing ..